sample.env

```diff
@@ -1,18 +1,13 @@
-# Copy this to .env and fill out values
-BASENAME=<insert name here>
-LOCATION='northcentralus'
+# Create a copy of this file called .env, and ensure that all variables have valid values.
 
-# Add values to configure the simulator (see README.md)
-SIMULATOR_MODE=generate
+# Deployment Config
+BASENAME=__CHANGE_ME___                                                         # Prefix for assets deployed into Azure
+LOCATION=__CHANGE_ME___                                                         # Azure region the assets will be deployed to 
 
-# Set this to False to return 404 for deployments that aren't configured
-ALLOW_UNDEFINED_OPENAI_DEPLOYMENTS=True
```

Comment:
Was this value intentionally removed?

Reply:
Yeah. The default is True, so I didn't see why this would be needed in the sample config.

Reply:
Are you OK with me removing this, given the default is already True?

Reply:
Yeah, let's leave it as-is 👍